conjure-java-jersey-server/build.gradle

```diff
@@ -20,15 +20,25 @@ dependencies {
     implementation 'javax.inject:javax.inject'
     implementation 'com.palantir.tritium:tritium-registry'
     implementation 'io.dropwizard.metrics:metrics-core'
+    implementation 'com.fasterxml.jackson.core:jackson-core'
+    implementation 'com.fasterxml.jackson.core:jackson-databind'
+    implementation 'com.fasterxml.jackson.dataformat:jackson-dataformat-cbor'
     implementation "com.fasterxml.jackson.jaxrs:jackson-jaxrs-cbor-provider", {
         // activation-api should be a compileOnly dependency where javax.activation should be used at runtime
         exclude module: 'javax.activation-api'
     }
+    implementation 'com.google.code.findbugs:jsr305'
+    implementation 'com.google.guava:guava'
     implementation "com.netflix.feign:feign-core"
     implementation "com.palantir.safe-logging:safe-logging"
     implementation "com.palantir.safe-logging:logger"
     implementation 'com.palantir.tokens:auth-tokens'
     implementation "com.palantir.tracing:tracing-jersey"
+    implementation 'org.glassfish.jersey.core:jersey-common', {
+        version {
+            strictly jerseyVersion
+        }
+    }
```

Comment:
Please double check that I didn't miss anything else for the legacy javax projects.